web/src/components/Content.tsx

```diff
@@ -54,18 +54,16 @@ export const Content = (props: ContentProps): ReactElement => {
     hr: (): ReactElement => {
       return <HorizontalLine />;
     },
-    blockquote: GreenBox,
     note: (props: any): ReactElement => {
       return <Alert variant="note">{props.children}</Alert>;
     },
     callout: (props: any): ReactElement => {
       return (
-        <Callout header={props.header} icon={props.icon} calloutType={props.calloutType}>
+        <Callout headerText={props.headerText} showIcon={props.showIcon} calloutType={props.calloutType}>
```

Comment:
pebble - is this missing the show header prop?

Reply:
Good catch!